test/shared/installationVerification.test.ts

```diff
@@ -475,29 +526,15 @@ describe('InstallationVerification Tests', () => {
       });
   });
 
-  it('NpmCommand Meta Request Error', async () => {
+  it('npm show fails to find a package', async () => {
     shelljsExecStub = getShelljsExecStub(sandbox, {} as NpmShowResults, 1, 'command execution error');
 
-    const iRequest: IRequest = (url: string, cb?: request.RequestCallback) => {
-      if (url.includes('sig')) {
-        cb(null, { statusCode: 200 } as request.Response, TEST_DATA_SIGNATURE);
-      } else if (url.includes('key')) {
-        cb(null, { statusCode: 200 } as request.Response, CERTIFICATE);
-      } else if (url.endsWith(plugin.name)) {
-        const err = new Error();
-        err.name = 'NPMMetaError';
-        cb(err, { statusCode: 500 } as request.Response, {});
-      } else {
-        throw new Error(`Unexpected test url - ${url}`);
-      }
-    };
-
```

Comment:
I think this UT was from when plugin-trust was doing raw HTTP calls to npm, we switched to npm here:
https://github.com/salesforcecli/plugin-trust/pull/154

so no need to stub `got`, the error is thrown from here:
https://github.com/salesforcecli/plugin-trust/blob/1affd57bd25563be4278c16b9249c8d765475ca6/src/shared/npmCommand.ts#L189